src/components/Catalog.vue

```diff
@@ -725,7 +745,14 @@ export default {
         this.assets && this.assets.length > 0 && "assets",
         this.zarrMetadataUrl && "zarrMetadata"
       ].filter(x => x != null && x !== false);
-    }
+    },
+    hasExternalPagination() {
```

Comment:
A shorter version would be :

`return Object.values(this.externalPaginationLinks).filter(link => link !== null).length > 0`